packages/core/lib/commands/test/index.js

```diff
@@ -123,6 +129,10 @@ const command = {
         option: "--grep|-g",
         description: "Use mocha's \"grep\" option while running tests. This " +
           "option only runs tests that match the supplied regex/string."
+      },
+      {
+        option: "--reporter|-r <name>",
+        description: "Specify the reporter type for mocha test. Default: 'spec'"
```

Comment:
This sentence sounds a bit odd. Maybe it would be better if it was something like, "Specify the type of mocha reporter to use during testing."?

Reply:
I think your sentence is sounding more proper @eggplantzzz . I will change it as you suggested.